src/components/orders-table/orders-data-table.component.tsx

```diff
@@ -45,6 +45,39 @@ interface OrdersDataTableProps {
   excludeCanceledAndDiscontinuedOrders?: boolean;
   useActivatedOnOrAfterDateFilter?: boolean;
 }
+interface CustomOrder extends Omit<Order, "orderer" | "patient"> {
+  patient: string;
+  patientUuid: string;
+  orderer: string;
+}
+
+function useSearchResults(tableEntries: Order[], searchString): CustomOrder[] {
```

Comment:
```suggestion
function useOrdersSearchResults(orders: Order[], searchTerm): CustomOrder[] {
```

---
Can you define this hook in a separate file?

Reply:
Thanks, forgot to relocate this